tracking/GenAI_Example.py

```diff
@@ -7,7 +7,7 @@
 # COMMAND ----------
 
 # MAGIC %pip uninstall -y mlflow-skinny
-# MAGIC %pip install git+https://github.com/mlflow/mlflow.git@mlflow-3 langchain langchain-databricks langchain-community databricks-agents
+# MAGIC %pip install git+https://github.com/mlflow/mlflow.git@mlflow-3 langchain databricks-langchain langchain-community databricks-agents
 # MAGIC dbutils.library.restartPython()
 
 # COMMAND ----------
```

Comment:
not sure why github wont let me select this line -- we also need to update L16 to say `from databricks_langchain`